### DISCUSSION TITLE
shelly 1 plus bluetooth

### DISCUSSION THREAD

**Discussion by ShifuSonny** [indomus/forum#1808] [GENERALE - Discussioni generiche]
Buongiorno,
Sono nuovo del forum e spero di scrivere nella sezione corretta...

Detto cio veniamo al sodo! Lo Shelly 1 plus ha nelle specifiche la connessione bluetooth per il paring ma non si può usare per connettere altri device?
Mi spiego, adesso che HA gestisce il bluetooth non sarebbe possibile collegare un dispositivo allo shelly 1 plus e poi inviare i dati via wifi a HA?
Sarebbe perfetto ad esempio per mi flora, o qualsiasi altro device bluetooth, che difficilmente è vicino ad HA ma potrebbe essere vicino ad uno shelly.

Grazie per le risposte!!

**Comment by snakuzzo** [comment]
Il BT con firmware nativo serve solo per il pairing. Se vuoi usarlo per altro dovresti usare un fw alternativo...tipo esphome

**Reply by ShifuSonny** [reply]
Grazie mille! Bella dritta! ho visto che con ESPhome ci sono già progetti per integrare qualsiasi dispositivo bl! Grazie mille di nuovo!

**Reply by NdK73** [reply]
Consiglio di configurare sempre il bottone di restart in esphome soprattutto quando si configura il BTLE: mi è capitato parecchie volte di dover staccargli manualmente l'alimentazione... Poi magari gli Shelly hanno meno problemi, ma non ho ancora provato i nuovi (ne ho molti dei vecchi ancora da montare). 
Tieni anche presente il limite di 4 connessioni contemporanee: se devi solo ricevere non hai problemi, ma p.e. certi tag richiedono la connessione :(

**Reply by ShifuSonny** [reply]
Grazie del consiglio! Purtroppo non sono skillatissimo con i firmware quindi mi ci vorrà un bel po' nel caso chiederò di nuovo aiuto! Pero pensavo fosse una cosa molto più complessa onestamente. In fin dei conti è solo una questione di programmazione base se non ho capito male.
